### PR TITLE
Add index of howtos and quicklab steps for ODH.

### PR DESCRIPTION
**Pull request by martinpovolny**
* add some info on installing ODH in Quicklab
* add index document content

read here: https://github.com/martinpovolny/continuous-deployment/blob/odh-install-quicklab/docs/index.md to review

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 Here you will find a series of docs that outline various procedures and how-tos
```
Comment (tumido) [inline docs/index.md:1]: Can you please mention the [`PV` setup docs](https://github.com/operate-first/continuous-deployment/blob/master/docs/downstream/on-cluster-persistent-storage/README.md) in here as well?

**Review by tumido — COMMENTED** [review]
One link is missing after a recent merge. Otherwise LGTM :+1:

**Comment by martinpovolny** [comment]
Thx, @tumido , added.

**Review by tumido — APPROVED** [review]

**Pull request merged**